qa/tasks/cram.py

```diff
@@ -124,7 +126,7 @@ def _run_tests(ctx, role):
     :param ctx: Context
     :param role: Roles
     """
-    assert isinstance(role, basestring)
+    assert isinstance(role, six.string_types)
```

Comment:
I hope we can do `isinstance(role, str)` I would prefer to use six as less as possible.

Reply:
Why? `six` is exactly there for theses cases. Imo there is no reason to avoid it (given that we already have it in the teuthology install_requires). Plus it's then easy to find the py2/py3 compat code if we are going to remove the python2 part.

Reply:
No longer part of this PR so marking this as resolved